tests/requirements.txt

```diff
@@ -1,3 +1,4 @@
+boto3==1.26.110
```

Comment:
Did we add something which uses boto3 and scikit-image in this PR or is it a fix for something which was failing before this PR?

Reply:
This is needed for the s3 bucket how-to guide